content/blog/2020-04-06-april-20-dvc-heartbeat.md

```diff
@@ -15,12 +15,11 @@ descriptionLong: |
   science.
 picture: 2020-04-06/april_header.png
 pictureComment:
-  A view from [Barrancas del
-  Cobre](https://en.wikipedia.org/wiki/Copper_Canyon), shot by Jorge Orpinel
-  Pérez. Jorge has mastered the art of working on DVC remotely.
-author: elle_obrien
-commentsUrl: https://discuss.dvc.org/t/april-20-heartbeat/347
-tags:
+  A view from
+  [Barrancas del Cobre](https://en.wikipedia.org/wiki/Copper_Canyon), shot by
+  Jorge Orpinel Pérez. Jorge has mastered the art of working on DVC remotely.
+  author: elle_obrien commentsUrl:
+  https://discuss.dvc.org/t/april-20-heartbeat/347 tags:
```

Comment:
something funny is happening here with tags

Reply:
Indeed. My bad (VSCode paragraph wrap). Fixed.